src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -258,12 +258,13 @@ type BackgroundServiceServer(state: State, client: FsacClient) =
             match msg, lst with
             //Empty
             | None, [] ->
-                checker.ClearLanguageServiceRootCachesAndCollectAndFinalizeAllTransients ()
+
+                // checker.ClearLanguageServiceRootCachesAndCollectAndFinalizeAllTransients ()
```

Comment:
I commented this out but we should probably remove it if this is a desired change.